committee-steering/sig-governance.md

```diff
@@ -0,0 +1,533 @@
+# SIG Governance Template
+
+## Goals
+
+The following document outlines a template governance structure for
+SIGs to follow.  Its goals are to provide the necessary structure
+for SIGs to self organize and self govern.
+
+This document is focussed on:
+
+- Outlining organizational responsibilities
+- Outlining organizational roles
+- Outlining processes and tools
+
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of project leadership
+- The process for making technical decisions
+- The process for ensuring continued test health
+
+## How to use this template
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+**Note:** Specific responsibilities towards the project are required
+some SIGs - e.g. ensuring test health.
+
+FAQ:
+
+- [Why is it so long?](#why-is-it-so-long)
+- [Why so much structure?](#why-so-much-structure)
+- [Why use mailing lists for tracking roles?](#why-use-mailing-lists-for-tracking-roles)
+- [Why track membership as a special role?](#why-track-membership-as-a-special-role)
+- [How do the results from the survey fit in?](#how-do-the-results-from-the-survey-fit-in)
+
+---
+
+# SIG ~~Foo~~ Governance
```

Comment:
A missing piece here is sub-projects.  Perhaps have that as an optional module SIGs can adopt?  (Could come in future PR).  Not clear who has responsibility to approve/charter sub projects and how those projects evolve.  Do sub-project members automatically become sig members?

Reply:
Other missing piece is how this charter gets updated.  Lazy consensus from members?

Reply:
RE: Subprojects - do you think that is something we can tackle after we have a baseline?

RE: Charter update: done - made it lazy consensus.  Should we also allow a 2/3 majority vote or only lazy consensus?